src/CalcViewModel/Common/LocalizationSettings.h

```diff
@@ -39,7 +41,7 @@ namespace CalculatorApp
                     result = GetLocaleInfoEx(m_resolvedName,
                         LOCALE_SDECIMAL,
                         decimalString,
-                        ARRAYSIZE(decimalString));
+                        std::size(decimalString));
```

Comment:
This (and others in this file/change) introduce new compilation warnings.  #197 turns on 'Treat warnings as errors' and then this will be a problem.

`1>calculator\src\calcviewmodel\common\localizationsettings.h(44): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of data (compiling source file Common\EngineResourceProvider.cpp)`

Resolve with `static_cast`.